File: Invertir_numero.py
def invertir(numero, invertido):
 
    (cociente, residuo) = divmod(numero, 10)
 
    if 0 == cociente:
        return invertido * 10 + residuo
    elif 0 == invertido:
        return invertir(cociente, residuo)
    else:
        return invertir(cociente, invertido * 10 + residuo)
 
 
numero = 12345
print(f"numero: {numero}, invertido: {invertir(numero, 0)}")

File: Invertir_numero.js
  

   function inverse(num) {
    let reversed = 0;
    
    
    while(num > 0) {
        reversed = reversed * 10;
        reversed = reversed + (num % 10);
        num = parseInt(num / 10);
    
}
alert( 'el numero 12345 Invertido es: '+ reversed);
}

inverse(12345);